README.md

```diff
@@ -96,6 +96,13 @@ This release changes the way in which location data is collected to better align
 3. Plug in your phone. Check it's detected with `adb devices`
 4. Execute: `make` (will also push app unto phone)
 
+# Testing - Instrumentation Tests (UI Tests)
+
+1. Install Android SDK
+2. Clone the repo
+3. Plug in your phone. Check it's detected with `adb devices`
+4. Execute: `./gradlew connected[Flavor]WebviewDebugAndroidTest` . Eg `./gradlew connectedUnbrandedWebviewDebugAndroidTest` or `./gradlew connectedMedicmobilegammaWebviewDebugAndroidTest`. At the moment we have tests only in these 2 flavors: unbranded and medicmobilegamma.
+
```

Comment:
Can we add here that to avoid failures running the tests, previous versions of the app should be uninstalled first, otherwise an `InstallException: INSTALL_FAILED_VERSION_DOWNGRADE` can make the tests to fail, and that Android needs to have English as default language, otherwise the tests fail with the following:

```
org.medicmobile.webapp.mobile.SettingsDialogActivityTest > testLoginScreen[SM-G986U1 - 11] FAILED 
        androidx.test.espresso.base.DefaultFailureHandler$AssertionFailedWithCauseError: 'a string containing "español"' doesn't match: Español (Spanish)
```